schemas/cs3/tbl/entry.ksy

```diff
@@ -36,6 +37,7 @@ seq:
         '"QSMons"': qs_mons
         "'SlotCost'": slot_cost
         "'SlotEp'": slot_ep
+        "'magicbo'": magic_bo
```

Comment:
Same as above

Reply:
The file should be renamed to magicbo since we try to match the table header name (except where it's CamelCase)